### PR TITLE
tweak: defund from fum

### PR DESCRIPTION
**Pull request by albertocuestacanada**

### CODE REVIEW

File: contracts/USM.sol [hunk 1]
```diff
@@ -72,11 +72,21 @@ contract USM is IUSM, Oracle, ERC20Permit, WithOptOut, Delegable {
         WithOptOut(optedOut_)
     {
         oracle = oracle_;
-        fum = new FUM(this, optedOut_);
+        FUM fum_ = fum = new FUM(this, optedOut_);
+        addDelegate(address(fum_));
```
Comment (jacob-eliosoff) [inline contracts/USM.sol:76]: Do you need this *and* the `msg.sender == address(fum)` condition below?

**Review by jacob-eliosoff — CHANGES_REQUESTED** [review]
Up to you...  If it seems safe I do think this cleans up our interface slightly.  But if we're in any doubt let's leave it as is.  Thanks though sir

File: contracts/USM.sol (hunk 1 above)
Comment (albertocuestacanada) [reply contracts/USM.sol:76]: The constructor needs to be reverted, sorry.

File: contracts/USM.sol [hunk 2]
```diff
@@ -121,25 +131,9 @@ contract USM is IUSM, Oracle, ERC20Permit, WithOptOut, Delegable {
...
-     * @notice Defunds the pool by redeeming FUM from an arbitrary address in exchange for equivalent ETH from the pool.
-     * Called only by the FUM contract, when FUM is sent to it.
-     * @param from address to deduct the FUM from.
-     * @param to address to send the ETH to.
-     * @param fumToBurn Amount of FUM to burn.
-     * @param minEthOut Minimum accepted ETH for a successful defund.
-     */
-    function defundFromFUM(address from, address payable to, uint fumToBurn, uint minEthOut)
-        external override
+        onlyHolderOrDelegateOrFUM(from, "Only holder or delegate")
```
[the remark this answers is not in the document]
Comment (albertocuestacanada) [reply contracts/USM.sol:134]: Yeah, right

**Comment by albertocuestacanada** [comment]
If you like it we can use it, it's safe.

**Pull request merged**